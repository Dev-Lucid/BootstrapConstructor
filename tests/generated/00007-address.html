<address></address>
<address><strong>test address1</strong><br />100 n. main st<br />apt 3<br />anywhere, new york 12345<br />T: 111.222.3333<br />F: 444.555.6666<br /><i class="icon-envelope"></i><a href="mailTo:testemail@testemail.notreal">testemail@testemail.notreal</a><br /></address>
<address></address>
<address></address>
