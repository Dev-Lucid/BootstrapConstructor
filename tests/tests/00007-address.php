<?php
$html = '';

$html .= bsc::construct('address')->render()."\n";
$html .= bsc::construct('address')
	->title('test address1')
	->address1('100 n. main st')
	->address2('apt 3')
	->city('anywhere')
	->region('new york')
	->postal_code('12345')
	->phone('111.222.3333')
	->fax('444.555.6666')
	->email('testemail@testemail.notreal')
	->render()."\n";
$html .= bsc::construct('address')->render(array(
	'address1'=>'200 s. main st',
	'city'=>'nowhere',
	'region'=>'california',
))."\n";
$html .= bsc::construct('address')
	->option('fields',array(
		'address1'=>'test_address_1',
		'city'=>'test_municipality',
		'region'=>'test_state',
	))->render(array(
		'test_address_1'=>'100 Everywhere st',
		'test_municipality'=>'Detroit',
		'test_state'=>'Michigan',
))."\n";


file_put_contents($output_path,$html);
?>