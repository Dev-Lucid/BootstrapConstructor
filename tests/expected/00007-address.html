<address></address>
<address><strong>test address1</strong><br />100 n. main st<br />apt 3<br />anywhere, new york 12345<br />T: 111.222.3333<br />F: 444.555.6666<br /><i class="icon-envelope"></i><a href="mailTo:testemail@testemail.notreal">testemail@testemail.notreal</a><br /></address>
<address>200 s. main st<br />nowhere, california<br /></address>
<address>100 Everywhere st<br />Detroit, Michigan<br /></address>
